android/demo-app/src/main/java/com/stadiamaps/ferrostar/AppModule.kt

```diff
@@ -59,7 +77,7 @@ object AppModule {
                 NavigationControllerConfig(
                     StepAdvanceMode.RelativeLineStringDistance(
                         minimumHorizontalAccuracy = 25U, automaticAdvanceDistance = 10U),
-                    RouteDeviationTracking.StaticThreshold(25U, 10.0),
+                    RouteDeviationTracking.StaticThreshold(15U, 25.0),
```

Comment:
Good reminder here to also write up ideal route deviation setups on iOS & Android. I went back to iOS for the first time in probably months and was appalled about how bad my setting was versus how well android was working 🤣

Reply:
Oh, interesting! Actually yeah I don't know since I have my own iOS demo app now haha. It's also tricky because on iOS you have pretty standard hardware with reasonably good geolocation services, whereas on Android, well, I'm intentionally testing on something with very bad geolocation ;) 

These few PRs as well as the not-yet-PR'd `autocomplete-search` branch will make the demo app _actually_ usable so that we can all improve the defaults rather than hacking up stuff in our own separate apps.